src/actions/testActions.ts

```diff
@@ -162,19 +165,21 @@ export const startTests =
             ', and that it uses firmware compatible with Direct Test Mode.';
 
         try {
-            dtm = new DTM(port, baudRate);
+            await updateDTM({ port: port ?? undefined, baudRate });
```

Comment:
Why the `?? undefined`?
Port is cast to string so shouldn't give you type errors.
Unless we get a device without any serialports, this will always be defined.
If you however do want to handle this case, it should probably just reject if port is undefined, a few lines above.